core/src/serve_repair.rs

```diff
@@ -76,6 +76,8 @@ pub const MAX_ANCESTOR_RESPONSES: usize =
 pub(crate) const REPAIR_PING_TOKEN_SIZE: usize = HASH_BYTES;
 pub const REPAIR_PING_CACHE_CAPACITY: usize = 65536;
 pub const REPAIR_PING_CACHE_TTL: Duration = Duration::from_secs(1280);
+// Rate limit to 4x the repair request iteration delay
+const REPAIR_PING_CACHE_RATE_LIMIT_DELAY: Duration = Duration::from_millis(4 * REPAIR_MS);
```

Comment:
400ms here sounds very low to me. the packet has to do a round trip and be processed at both nodes.

Also what is the reasoning to define this in terms of `REPAIR_MS`? i.e why should changing `REPAIR_MS` change this value too?

Reply:
I decoupled the delay from `REPAIR_MS`. I'm using a small multiple of `REPAIR_MS` because the repair request side is only waiting `REPAIR_MS` before sending the next batch of repair requests. If a ping or pong is lost the request side will not be able to successfully get a repair processed until the `rate_limit_delay` expires.

Reply:
yeah, but if that is like a few seconds it shouldn't do much harm and also the delay should at least be long enough for:
* the ping is serialized and sent to the socket.
* the packet travels to the other node
* the other node processes the packet, and sends back the pong response
* the packet travels back to this node
* this node processes the packet and register the response.

Otherwise each time we are sending 2 ping packets.
Picking the delay shorter than above, wouldn't make things faster; it would only add overhead.

US-Europe round-trip traversal for a packet seems to be ~400ms at least based on some numbers I had seen previously. And that is just the packet traversal and not accounting for any processing latencies.

I think we can set this delay to >= 2 seconds or something like that.

Reply:
Increased to 2s.